### DISCUSSION TITLE
Include unrealized gains in backtesting result

### DISCUSSION THREAD

**Discussion by amustapha** [coding-kitties/investing-algorithm-framework#280] [Ideas]
After running backtesting on a strategy, we get results that look like the screenshot above with orders that haven't been closed showing a net profit of 0. It might be quite helpful to show the floating (unrealized) profit or loss at the time when the backtesting finished.
<img width="1168" alt="image" src="https://github.com/user-attachments/assets/42c7624b-cd9a-4812-af72-7fe8a8cdb848" />

**Comment by MDUYN** [comment]
Hey @amustapha, good point, I created an issue for this. Thanks!

**Reply by MDUYN** [reply]
https://github.com/coding-kitties/investing-algorithm-framework/issues/286
